packages/ember-htmlbars/tests/compat/handlebars_get_test.js

```diff
@@ -34,13 +34,13 @@ QUnit.module("ember-htmlbars: compat - Ember.Handlebars.get", {
 QUnit.test('it can lookup a path from the current context', function() {
   expect(1);
 
-  registry.register('helper:handlebars-get', function(path, options) {
+  registry.register('helper:handlebars-get', new HandlebarsCompatibleHelper(function(path, options) {
```

Comment:
After discussing with @mixonic, this is needed because we are wrapping eagerly when a helper is resolved (in the resolver changes in this PR). In this case, we are bypassing the resolver so this wrapping is required.